examples/js/loaders/STLLoader.js

```diff
@@ -164,34 +163,28 @@ THREE.STLLoader.prototype = {
 
 				var vertexstart = start + i * 12;
 
-				vertices[ offset ] = reader.getFloat32( vertexstart, true );
-				vertices[ offset + 1 ] = reader.getFloat32( vertexstart + 4, true );
-				vertices[ offset + 2 ] = reader.getFloat32( vertexstart + 8, true );
+				vertices.push( reader.getFloat32( vertexstart, true ) );
+				vertices.push( reader.getFloat32( vertexstart + 4, true ) );
+				vertices.push( reader.getFloat32( vertexstart + 8, true ) );
 
-				normals[ offset ] = normalX;
-				normals[ offset + 1 ] = normalY;
-				normals[ offset + 2 ] = normalZ;
+				normals.push( normalX, normalY, normalZ );
 
 				if ( hasColors ) {
 
-					colors[ offset ] = r;
-					colors[ offset + 1 ] = g;
-					colors[ offset + 2 ] = b;
+					colors.push( r, g, b );
 
 				}
 
-				offset += 3;
-
 			}
 
 		}
 
-		geometry.addAttribute( 'position', new THREE.BufferAttribute( vertices, 3 ) );
```

Comment:
Or, if we're keeping them, we should make them look like the rest of the API by doing...

```js
function Float32Attribute( array, itemSize ) {

	return new BufferAttribute( new Float32Array( array ), itemSize );

}

Float32Attribute.prototype = InstancedBufferAttribute.prototype = Object.create( BufferAttribute.prototype );
Float32Attribute.prototype.constructor = Float32Attribute;
```